cities/usingen.json

```diff
@@ -0,0 +1,26 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                  8.53383,
+                  50.33561
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Usinger Wochenmarkt",
+                "location": "Alter Marktplatz, Usingen",
+                "opening_hours": "Fr 08:00-18:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Usingen",
```

Comment:
Please note that the [contribution guide](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/CONTRIBUTING.md) recommends to add a license and an update date here, e.g. `"title": "Stadt Berlin, CC BY 3.0 DE, aktualisiert am 03.05.2019",`